packages/daemon/src/wallet/LedgerClient.js

```diff
@@ -0,0 +1,65 @@
+// @flow
```

Comment:
Again just to follow JS conventions, could this file have a lowercase `ledger` please so we differentiate from the other modules that export classes?